contracts/UFragmentsPolicy.sol

```diff
@@ -177,8 +191,11 @@ contract UFragmentsPolicy is Ownable {
     {
         Ownable.initialize(owner);
 
-        deviationThreshold = (5 * TARGET_RATE) / 100;  // 5% of target
-        minimumVolume = 1;
+        targetRate = INITIAL_CPI.mul(10 ** RATE_DECIMALS).div(TARGET_CPI);
+
+        // 5.0%, RATE_DECIMALS Fixed point
+        deviationThresholdPerc = 5 * 10 ** (RATE_DECIMALS);
```

Comment:
I think it is cleaner to store it as a fraction not a percentage, so it become 0.05 and removing div(100) at use time

Reply:
Let's discuss this